common/hooks/tx/use-escrow-swap.ts

```diff
@@ -18,6 +18,9 @@ export const useEscrowSwap = (swap: InboundSwapSent) => {
     const txData = await fetchTxData(btcTxid, address);
     const hash = hashSha256(hexToBytes(secret));
     const swapperHex = numberToLE(swapperId);
+    const amount = txData.amount;
+    const amountWithFeeRate = (amount * (10000n - intToBigInt(supplier.inboundFee))) / 10000n;
+    const minToReceive = amountWithFeeRate - intToBigInt(supplier.inboundBaseFee);
```

Comment:
Also just an FYI, since we know this is a number, you can just use `BigInt(supplier.inboundBaseFee)`. Again, not a big deal. The `intToBigInt` function is mainly very handy when the type is something like `string | number | bigint`